addons/prometheus/0.38.x/prometheus-2.yaml

```diff
@@ -22,9 +22,10 @@ metadata:
     docs.kubeaddons.mesosphere.io/grafana: "https://grafana.com/docs/"
     docs.kubeaddons.mesosphere.io/alertmanager: "https://prometheus.io/docs/alerting/alertmanager/"
     values.chart.helm.kubeaddons.mesosphere.io/prometheus: "https://raw.githubusercontent.com/mesosphere/charts/81ca01f13e88fc05bf195c95356a9cfd8902ed86/staging/prometheus-operator/values.yaml"
-    # The prometheus-operator chart from prior Konvoy releases can't be upgraded to this chart version.
-    # See https://jira.d2iq.com/browse/DCOS-62924.
-    helmv2.kubeaddons.mesosphere.io/upgrade-strategy: '[{"upgradeFrom": "<=5.19.7", "strategy": "delete"}]'
+    # The prometheus-operator chart from prior Konvoy releases can't be upgraded to 8.10.0.
+    # See https://github.com/helm/charts/issues/21200.
+    # 8.8.5 was the latest version available in mesosphere/charts before it was bumped past 8.10.0.
+    helmv2.kubeaddons.mesosphere.io/upgrade-strategy: '[{"upgradeFrom": "<=8.8.5", "strategy": "delete"}]'
```

Comment:
we've employed this strategy twice now with the prom-operator (both from major and minor upgrades). I wonder if it would make more sense to change it from `<=8.x` to `>=5.19.7`? Otherwise we'll have to keep updating this strategy version.

Reply:
The goal is to _not_ delete first. It's more efficient.